### DISCUSSION TITLE
Markdown and Latex and Variables

### DISCUSSION THREAD

**Discussion by dorn-gerhard** [fonsp/Pluto.jl#995] [Questions about using Pluto]
I am not yet so familiar with Markdown but would like to ask 
what's the best strategy to combine Markdown with variables or Latex.

e.g

    begin
      result = 4
      md"""
      Surface = $(result) $m^2$
      """
    end

The result is
 
    Surface = (*result*) = m^2$

since $(result) $ was interpreted as Latex and not as Julia variable. 
Is there a workaround?

Similar problems arise when using Latex or variables inside tables or other Markdown objects

**Comment by paradocs** [comment]
@dorn-gerhard
I have fought the $$ rabbit hole with little luck.  
But for units, use the wonderful Julia  unit macros.

``Pkg.add("Unitful")``
``using Unitful``

Just follow a number with the letter  u   
and a double quoted unit dimension.

result = 4 m^2
``result =  2u"m" * 2u"m"``

Surface = 4 m^2
``md"""
Surface = $result
"""``
There is even UnitfulUS with miles and gallons!
----- EDIT March 22 2021
1) There is a  way in Pluto markdown to mix
$variables with something similar to LaTex.
Within md"""    start and stop the math formatting
with two backticks  ``
On my keyboard it is a deadkey for accent marks
so I must press twice to get each backtick.
They are small so look carefully to find the key.
2) My examples may be non-sense or non standard.
3) Please include anti-gambling education.
https://www.geckoandfly.com/27973/anti-gambling-addiction-quotes/
   
``result1 = ["red ", "green ", "red ", "black "]``
``result2 = ["♣", "♥", "♠", "♦"]``
``sliderResult  = result2[2]``

code>
md"""
n choose x \`\`~~~~\`\` \`\` n \choose x \`\` \`\`~~~~\`\` $result1

Here is my lucky card draw: $result2  

  \`\`_nC^k = \binom nk=\frac{n!}{k!(n-k)!}\`\`

  \`\`_nP^k = \frac{n!}{(n-k)!}\quad\`\`

And the winner is: $sliderResult
"""
<code
